talonone-demo/management-api.ts

```diff
@@ -0,0 +1,50 @@
+import * as TalonOne from "talon_one";
+import dotenv from "dotenv";
+
+const defaultClient = TalonOne.ApiClient.instance;
+defaultClient.basePath = process.env.TALON_ONE_BASEPATH;
+
+const api = new TalonOne.ManagementApi();
+
+export const CreateAttributes = async () => {
+  const authBody = {
+    email: process.env.TALON_ONE_ADMIN_EMAIL,
+    password: process.env.TALON_ONE_PASSWORD,
+  };
+  console.log('authBody', authBody);
+
+  const res = await api.createSession(authBody);
+  const resJson = await res;
```

Comment:
Did you mean to do this?
```suggestion
  const resJson = await res.json();
```

because if not this `await` is redundant with the `await` on the line above it.

Reply:
parsed object was coming and I forgot to remove the line, tested it and it looks good without